### PR TITLE
code-minimap: init at 0.4.3

### PR DESCRIPTION
**Pull request by andir**
I'm forwarding this patch that I received via email:

```
commit ff0b90414ae7064e1ad37aff865f99e82bbee3e5
Author: Ben Sima <ben@bsima.me>
Date:   Fri Dec 25 00:22:34 2020 -0500

    code-minimap: init at 0.4.3

    Message-Id: <20201225052234.17216-1-ben@bsima.me>
```

You can find the submission in the [archive](https://lists.sr.ht/~andir/nixpkgs-dev/%3C20201225052234.17216-1-ben%40bsima.me%3E).

### CODE REVIEW

File: pkgs/tools/misc/code-minimap/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    owner = "wfxr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "03azqy4i15kfpd0gzjaw2di9xva4xdf95yb65b93z3y9y5wy4krc";
+  };
+
+  cargoSha256 = "1rxrdavj07i7qa5rf1i3aj7zdcp7c6lrg8yiy75r6lm4g98izzww";
+
+  meta = with stdenv.lib; {
+    description = "A high performance code minimap render.";
```
Comment (fabaff) [inline pkgs/tools/misc/code-minimap/default.nix:20]:
```suggestion
    description = "A high performance code minimap render";
```

see https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 107563` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>code-minimap</li>
  </ul>
</details>

**Comment by andir** [comment]
Thanks.

> ```suggestion
>     description = "A high performance code minimap render";
> ```

Fixed the trailing period.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107563` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>code-minimap</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107563` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>code-minimap</li>
  </ul>
</details>

**Pull request merged**